### PR TITLE
Dev > Prod

### PR DESCRIPTION
**Pull request by jkonrath-postman**
TW-1726 Rename Insights folder and file names #5431
TW-1702 Approving requests for access - expiration deadline
TW-1553 Create overview for /docs/writing-scripts/script-references #5378

### CODE REVIEW

**Review by nerminamiller-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/insights/insights-troubleshoot.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Here are a few things that may be going on:
 * Permission issues that prevent the LCA from seeing traffic.
 * Encrypted traffic. See [What do I do if my traffic is encrypted?](#what-do-i-do-if-my-traffic-is-encrypted) for solutions.
 * Data formats that the LCA doesn't recognize.
-* ECS on EC2 with bridge networking. You can install LCA as a daemon service only. For instructions, see the [installation instructions](/docs/live-insights/live-insights-gs/#install-lca-as-a-daemon-service).
+* ECS on EC2 with bridge networking. You can install LCA as a daemon service only. For instructions, see the [installation instructions](/docs/live-insights/insights-gs/#install-lca-as-a-daemon-service).
```
Comment (mark-anthony-postman) [inline src/pages/docs/insights/insights-troubleshoot.md:115]:
```suggestion
* ECS on EC2 with bridge networking. You can install LCA as a daemon service only. For instructions, see the [installation instructions](/docs/insights/insights-gs/#configure-lca-as-a-daemon-service).
```

**Review by mark-anthony-postman — APPROVED** [review]
Found one broken link, otherwise LGTM!

File: src/pages/docs/insights/insights-troubleshoot.md (hunk 1 above)
Comment (nerminamiller-postman) [reply src/pages/docs/insights/insights-troubleshoot.md:115]: Thank you, @mark-anthony-postman!

**Pull request merged**